lldb/source/Core/Progress.cpp

```diff
@@ -16,9 +16,10 @@ using namespace lldb_private;
 
 std::atomic<uint64_t> Progress::g_id(0);
 
-Progress::Progress(std::string title, uint64_t total,
+Progress::Progress(std::string title, bool is_discrete, uint64_t total,
                    lldb_private::Debugger *debugger)
-    : m_title(title), m_id(++g_id), m_completed(0), m_total(total) {
+    : m_title(title), m_is_discrete(is_discrete), m_id(++g_id), m_completed(0),
+      m_total(total) {
```

Comment:
Does this introduce a warning with `-Wreorder-ctor`?

Reply:
It did initially but I reordered the variable declarations in the header in order to match how they are in the constructor.